BLSQ/iaso
# POLIO-548: view campaigns logs details

Explain what problem this PR is resolving

There was yet no way for the user to see which modifications have been done on a campaign

## Self proof reading checklist

## Changes

- add link for campaign log history in `CreateEditDialog`
- add new component CampaignLogDetail to show log details
- add a link to logs api if value is an object (eg. rounds)
- add new route for campaign log history component : `CampaignLogDetail`

## How to test

1. Go to campaigns with a polio user
2. Edit a campaign 
3. In the `CreateEditDialog` you should be able to see a history icon at the upper right corner
4. Click on history icon 
5.  You can select a log and see the changes


## Print screen / video

[recording-2022-11-16-11-28-20.webm](https://user-images.githubusercontent.com/25134301/202156404-23e64215-f508-4c24-9ac5-8ed88cbb8ce2.webm)
